### ISSUE TITLE
Ensure bike attributes can be updated via the API

### ISSUE THREAD

**Issue by sethherr** [bikeindex/bike_index#955]
Specific attributes we've been told are not updating:

name, other serial or registration number, accessories, components, colors

**Comment by jmromer** [comment]
Documenting...

```
PUT /v3/bikes/{id}
  "error": "OAuth error: Permission to *update* the bikes you have on Bike Index",
```
```
GET /v3/me/bikes
  "error": "OAuth error: Permission to see what bikes you have on Bike Index",

```

**Comment by jmromer** [comment]
> name

`name` is not documented as updateable -- do we want it to be? (I can't reproduce the bug with component manufacturer names)

> other serial or registration number

Likewise with component serial

> accessories
> components

I haven't been able to reproduce this with any accessory or component fields

> colors

colors: Should be addressed by #960 

I wonder also if the UI around creating a new application via the UI might be misleading some people? (i.e., they might be using the default access token, which has no permissions, and missing how to create a new one since you have to delete the existing one to create a new one.)

**Comment by sethherr** [comment]
I'm marking this resolved. I think it's good enough.
